pkg/util/utils.go

```diff
@@ -297,6 +297,9 @@ func GetDefaultStoreOptions() (storage.StoreOptions, error) {
 		err                      error
 	)
 	storageOpts := storage.DefaultStoreOptions
+	if !rootless.IsRootless() {
```

Comment:
This code makes little sense.  Above you set storageOpts to DefaultStoreOptions, then you overwrite it in both root and rootless mode.